airflow/providers/ssh/hooks/ssh.py

```diff
@@ -106,8 +106,11 @@ def __init__(
                     self.key_file = extra_options.get("key_file")
 
                 private_key = extra_options.get('private_key')
-                if private_key:
+                private_key_passphrase = extra_options.get('private_key_passphrase')
```

Comment:
Very minor thing (not a must-change): maybe code like below would be more straight forward from reading aspect?

```
if private_key:
    if private_key_passphrase:
        ...
    else:
        ...
```

Reply:
@XD-DENG Done, let me know what you think